modules/@apostrophecms/asset/lib/webpack/apos/webpack.config.js

```diff
@@ -86,3 +87,16 @@ module.exports = ({
 
   return merge(config, ...tasks);
 };
+
+function getVueAlias() {
+  if (!process.env.APOS_DEV) {
+    return '@vue/runtime-dom';
+  }
+
+  const vueProjectLevelPath = path.join(process.cwd(), 'node_modules/@vue/runtime-dom');
```

Comment:
```suggestion
  const vueProjectLevelPath = path.resolve('./node_modules/@vue/runtime-dom');
```

☝️ does it work with npm links?

Reply:
I took this line, it should work because it target the current running project `node_modules` folder.